docs/tasks/tools/install-kubectl.md

```diff
@@ -120,7 +120,7 @@ kubectl is available as a [snap](https://snapcraft.io/) application.
        install-kubectl.ps1 [-DownloadLocation <path>]
        
 If no Downloadlocation is specified, kubectl will be installed in users temp Directory   
-2. The installer creates $HOME/.kube and instucts to create a config file  
+2. The installer creates $HOME/.kube and instructs to create a config file  
```

Comment:
The typo fix is good, but the resulting sentence doesn’t read correctly. Who or what is being instructed to create a configuration file? I think rephrasing this is needed

Reply:
Ok, I will solve this problem.

Reply:
This problem has been fixed, please review.